src/classes/DBConnect.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Store;
+
+use \PDO;
+use \Exception;
+
+class DBConnect {
```

Comment:
SOLID would have you name this mySqlDbConnect and have it implement an interface that ensures connectToDb

Reply:
also { on next line

Reply:
refactored